scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -275,7 +280,11 @@ def get_policy_name(application_name, pipeline_name, gate_name):
     try:
         data = application_name, pipeline_name, gate_name
         cur_oesdb.execute("select policy_name from policy_gate where application_name = %s and pipeline_name = %s and gate_name = %s", data)
-        return cur_oesdb.fetchone()[0]
+        policyName = cur_oesdb.fetchone()
+        if  policyName is not None:
+            return policyName[0]
+        else:
+            logging.info("Policy did not find for the  application name : %s, Pipeline name : %s, gate name : %s", application_name, pipeline_name,gate_name)
```

Comment:
change it to "policy name was not found for .... "

Reply:
updated the log message